sync/sync_test.go

```diff
@@ -115,7 +115,7 @@ func TestSyncBlocks(t *testing.T) {
 			case 2:
 				state.BlockHash = new(felt.Felt) // fail sanity checks
 			case 3:
-				state.OldRoot = new(felt.Felt).SetUint64(1) // fail store
+				state.NewRoot = new(felt.Felt).SetUint64(1) // fail store
```

Comment:
What is the reason for this change?

Reply:
I took the liberty to remove OldRoot check, since it is kinda redundant. `verifyBlock` already checks the parent block hash against the local `HEAD`, which already proves that our state root is `OldRoot`